src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -135,6 +135,7 @@ def run(input: str, config: str, rules: str, rejects: str, output: str):
         df=msdf.df, filter_prefixes=prefix_of_interest, features=[SUBJECT_ID, OBJECT_ID]
     )
     msdf.df[PREDICATE_MODIFIER] = "" if PREDICATE_MODIFIER not in msdf.df.columns else msdf.df[PREDICATE_MODIFIER]
+    mapping_msdf.df[PREDICATE_MODIFIER] = "" if PREDICATE_MODIFIER not in mapping_msdf.df.columns else mapping_msdf.df[PREDICATE_MODIFIER]
```

Comment:
I trust you on this, but it seems this should not be necessary. 

```
msdf.remove_mappings(mapping_msdf)
```

Should be resilient to data frames that don't have a `PREDICATE_MODIFIER` column!